test/integration/framework/perf_utils.go

```diff
@@ -73,8 +73,18 @@ func (p *IntegrationTestNodePreparer) PrepareNodes() error {
 		},
 	}
 	for i := 0; i < numNodes; i++ {
-		if _, err := p.client.CoreV1().Nodes().Create(baseNode); err != nil {
-			glog.Fatalf("Error creating node: %v", err)
+		retry := 0
```

Comment:
var err error
for retry = 0; retry < retries; retry++ {
  _, err = p.client....
  if err == nil || !e2eframework.IsRetryable... {
    break
  }
}
if err != nil {
  glog.Fatalf(...)
}


I think that ^^ is more clear.

Reply:
Nice suggestion! Done.